contracts/TxRelay.sol

```diff
@@ -26,8 +26,9 @@ contract TxRelay {
     ) public {
 
         address claimedSender = getAddress(data);
-        // relay :: nonce :: destination :: data :: relayer
-        bytes32 h = sha3(this, nonce[claimedSender], destination, data);
+        // use EIP 191
+        // 0x19 :: version :: relay :: nonce :: destination :: data :: relayer
```

Comment:
Since we're changing this code, it's worth updating the comment to remove "` :: relayer`" at the end, which is no longer part of the plaintext.